src/main/resources/sql/court/update-court.sql

```diff
@@ -0,0 +1 @@
+UPDATE court SET court_name=:name, phone=:phone,extension=:extension,website=:website,payment_system=:paymentSystem,address=:address,city=:city,state=:state,zip_code=:zipCode,latitude=:latitude,longitude=:longitude,citation_expires_after_days=:citationExpires WHERE court_id = :id;
```

Comment:
You shouldn't need this file. Spring Data should eliminate the need for this. Plus, nothing references this file anyway.

Reply:
Yes, this code was left over from pre-spring data conversion.